ui/tests/acceptance/oidc-auth-method-test.js

```diff
@@ -21,34 +22,27 @@ module('Acceptance | oidc auth method', function (hooks) {
     // ensure clean state
     localStorage.removeItem('selectedAuth');
   });
+
   hooks.afterEach(function () {
     this.openStub.restore();
   });
 
-  const login = async (select) => {
-    await visit('/vault/auth');
-    // select from dropdown or click auth path tab
-    if (select) {
-      await fillIn('[data-test-select="auth-method"]', 'oidc');
-    } else {
-      await click('[data-test-auth-method-link="oidc"]');
-    }
-    later(() => {
-      window.postMessage(buildMessage().data, window.origin);
-      cancelTimers();
-    }, 50);
-    await click('[data-test-auth-submit]');
-  };
-
   test('it should login with oidc when selected from auth methods dropdown', async function (assert) {
     assert.expect(1);
 
     this.server.get('/auth/token/lookup-self', (schema, req) => {
       assert.ok(true, 'request made to auth/token/lookup-self after oidc callback');
       req.passthrough();
     });
-
-    await login(true);
+    authPage.logout();
```

Comment:
Out of curiosity - did logging out in the afterEach hook not work here? 
```  
hooks.afterEach(function () {
    return authPage.logout();
  });
  ```

Reply:
Yeah, I tried that too. I didn't. I honestly don't know why this test was having issues with keeping the logged out between states.